templates/partials/community.html

```diff
@@ -1,16 +1,46 @@
 {% import "macros/icon.html" as icon_macros %}
 
 <h2 id="community">Community</h2>
+<dl class="features">
+  <div>
+    <dt>
+      <div>
+        {{ icon_macros::sprite_icon(name="discourse") }}
+        Users Forum
+      </div>
+    </dt>
+    <dd>For asking user questions related to Vala use the Discourse server with the Vala tag.</dd>
+    <dd><a href="{{ config.extra.socials.discourse }}">Discourse Forums</a></dd>
+  </div>
+  <div>
+    <dt>
+      <div>
+        {{ icon_macros::sprite_icon(name="matrix") }}
+        Internals Chat
+      </div>
+    </dt>
+    <dd>To talk to the developers or ask questions how to contribute to Vala use the Vala Matrix channel. It also has a IRC bridge.</dd>
+    <dd><a href="{{ config.extra.socials.matrix }}">Matrix Room</a></dd>
+  </div>
+  <div>
+    <dt>
+      <div>
+        {{ icon_macros::sprite_icon(name="discord") }}
+        Community Server
+      </div>
+    </dt>
+    <dd>To hang out with the community use the Discord server.</dd>
+    <dd><a href="{{ config.extra.socials.discord }}">Discord Server</a></dd>
+  </div>
+</dl>
```

Comment:
The wording needs some tweaks but this is great!

Reply:
Just push to the branch.

Reply:
https://docs.github.com/en/site-policy/github-terms/github-community-guidelines